opencog/reasoning/RuleEngine/rule-engine-src/JsonicControlPolicyParamLoader.cc

```diff
@@ -156,6 +172,25 @@ void JsonicControlPolicyParamLoader::read_null(const Value &v, int lev)
 }
 
 template<typename > void JsonicControlPolicyParamLoader::read_primitive(
-		const Value &v, int lev)
+        const Value &v, int lev)
 {
 }
+
+const string JsonicControlPolicyParamLoader::load_json_file_relative(
+        const string& filename, vector<string> search_paths)
+{
```

Comment:
Since this method is not loading, but checking/getting the path, I think it is better to rename it.